### PR TITLE
[PRM-686] Added remove button to uploadItem

### PR DESCRIPTION
**Pull request by JBlacklaws**

### CODE REVIEW

**Review by LewisMitchell — CHANGES_REQUESTED** [review]
Looks good, just some minor changes

**Comment by LewisMitchell** [comment]
image

File: app/views/components/upload/uploadItem.scala.html [hunk 1]
```diff
@@ -19,7 +19,11 @@
                 </div>
             </div>
 
-            @* TODO: add remove/change links *@
+            <button type="button" class="multi-file-upload__remove-item govuk-link">
+                @messages("otherReason.uploadEvidence.button.remove")
+                <span class="govuk-visually-hidden">@messages("otherReason.uploadEvidence.document") <span class="multi-file-upload__number">{fileNumber}</span></span>
```
[the remark this answers is not in the document]
Comment (JBlacklaws) [reply app/views/components/upload/uploadItem.scala.html:24]: otherReason.uploadEvidence.document is already in the message files, on line 293

[the remark this answers is not in the document]
Comment (LewisMitchell) [reply app/views/components/upload/uploadItem.scala.html:24]: Whoops! Great!

**Review by LewisMitchell — APPROVED** [review]

**Pull request merged**